### PR TITLE
Change/default ports in dev cli

### PR DESCRIPTION
**Pull request by frankcorneliusmartin**
Please review and merge #1427 first. This solves https://github.com/vantage6/vantage6-workshop/issues/66 and partially #1428

### CODE REVIEW

File: vantage6/vantage6/cli/dev/create.py [hunk 1]
```diff
@@ -476,19 +476,19 @@ def demo_network(
...
-    default=5001,
+    default=7600,
     help="Port to run the UI on. Default is 5001.",
 )
 @click.option(
     "--algorithm-store-port",
     type=int,
-    default=5002,
+    default=7602,
     help="Port to run the algorithm store on. Default is 5002.",
```
Comment (bartvanb) [inline vantage6/vantage6/cli/dev/create.py:492]: Help text should also be changed for each of the three port changes

**Review by bartvanb — APPROVED** [review]
I reviewed the other PR you mentioned but somehow the same changes are still shown here (I even tried to switch target branch which helped before but it didn't now). Anyhow, I had one comment, if you resolve that one, feel free to merge it

File: vantage6/vantage6/cli/dev/create.py (hunk 1 above)
Comment (frankcorneliusmartin) [reply vantage6/vantage6/cli/dev/create.py:492]: Done, thanks.

**Comment by frankcorneliusmartin** [comment]
> I reviewed the other PR you mentioned but somehow the same changes are still shown here (I even tried to switch target branch which helped before but it didn't now). Anyhow, I had one comment, if you resolve that one, feel free to merge it

I rebased, should be good to go now. @bartvanb I well let you merge it.

**Review by bartvanb — APPROVED** [review]

**Pull request merged**